### PR TITLE
Improve UI

### LINKED ISSUE #271: Improve message_success for totalorders command

### PR DESCRIPTION
**Pull request by GnohChengYi**
Fixes #271

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/TotalOrdersCommand.java [hunk 1]
```diff
@@ -14,7 +14,7 @@
             + "Example: " + COMMAND_WORD;
 
     public static final String SHOWING_TOTAL_ORDERS_MESSAGE = "Opened/Refreshed total orders window.\n"
-            + "You might need to click the icon on taskbar if you have minimized it.";
+            + "You might need to restore the window if you have minimized it.";
```
Comment (ngchisern) [inline src/main/java/seedu/address/logic/commands/TotalOrdersCommand.java:17]: You might want to add this message to the help command as well if you add it here.

**Review by ngchisern — CHANGES_REQUESTED** [review]
Added a minor suggestion below.

File: src/main/java/seedu/address/logic/commands/TotalOrdersCommand.java (hunk 1 above)
Comment (GnohChengYi) [reply src/main/java/seedu/address/logic/commands/TotalOrdersCommand.java:17]: Okay

**Review by ngchisern — APPROVED** [review]
LGTM

**Pull request merged**